extensions/sonarr/package.json

```
@@ -0,0 +1,92 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sonarr",
+  "title": "Sonarr",
+  "description": "View your Sonarr instance, upcoming shows and more",
+  "icon": "command-icon.png",
+  "author": "Aayush9029",
+  "categories": [
+    "Media",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Sonarr Shows",
+      "subtitle": "List of upcoming shows",
+      "description": "All upcoming shows date for 2 weeks from now",
```

Since the future days is configurable I would drop the 2 weeks from the description
```suggestion
      "description": "All upcoming shows dates",
```